modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -540,6 +540,23 @@ class Editor implements EditorObservable {
    * @param {String/Function} cmdFunc Command name string or function to execute when the key is pressed.
    * @param {Object} scope Optional scope to execute the function in.
    * @return {Boolean} true/false state if the shortcut was added or not.
+   * @example
+   * editor.addShortcut('ctrl+a', "description of the shortcut", function() {});
+   * editor.addShortcut('ctrl+alt+a', "description of the shortcut", function() {});
+   * editor.addShortcut('meta+a', "description of the shortcut", function() {});
+   * // "meta" maps to Command on Mac and Ctrl on PC
```

Comment:
having the comment on a new line after the line it refers to reads oddly to me? I may just used to comments being before/inline though

Reply:
Agreed... I thought this applied to the next line to be honest. These comments should be above the line, as that's the normal convention.

Reply:
Fixed by https://github.com/tinymce/tinymce/pull/5818/commits/8b8bd26d364e6dade5085a758b2bf0484967a07b